src/dotnet-install.ps1

```diff
@@ -204,6 +204,15 @@ function Get-Machine-Architecture() {
         return $ENV:PROCESSOR_ARCHITEW6432
     }
 
+    # covers the case when PS x64 is run on ARM machine
+    if( (wmic os get osarchitecture)[2] -like "ARM*" ) {
+        if( [Environment]::Is64BitOperatingSystem )
```

Comment:
the ternary operator was introduced in PS v7 https://toastit.dev/2019/09/25/ternary-operator-powershell-7/
I am afraid it can bring some problems with compatibility for some users